src/Ext/House/Body.cpp

```diff
@@ -622,3 +622,259 @@ DEFINE_HOOK(0x50114D, HouseClass_InitFromINI, 0x5)
 
 	return 0;
 }
+
+int CountOwnedIncludeDeploy(const HouseClass* pThis, const TechnoTypeClass* pItem)
```

Comment:
```suggestion
int HouseExt::CountOwnedIncludeDeploy(const HouseClass* pThis, const TechnoTypeClass* pItem)
```
Should it be like this?

Reply:
Now that all functions are moved into HouseClass, there's no need to make this a HouseExt function for outer call